src/main/java/seedu/address/model/util/Description.java

```diff
@@ -13,6 +13,7 @@
 public class Description {
 
     public static final String EMPTY_DESCRIPTION_FIELD = "-";
+    // TODO: remove after allowing blank optional fields to pass.
     public static final String MESSAGE_CONSTRAINTS = "Descriptions can take any values, and it should not be blank";
```

Comment:
missing period for constriants

Reply:
Message will be removed later on, so will not be making extra changes.